homeassistant/components/fyta/config_flow.py

```diff
@@ -31,14 +32,17 @@ class FytaConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a config flow for Fyta."""
 
     VERSION = 1
+    MINOR_VERSION = 1
+
     _entry: ConfigEntry | None = None
+    credentials: dict[str, str | datetime] = {}
```

Comment:
Don't define this dict at class level, define it in a constructor instead. Otherwise this will have side effects which are hard to debug